src/app/personside/visittkort-v2/header/FeilendeSystemerModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { InformasjonElement } from '../PersondataDomain';
+
+const Modal = styled(RawModal)`
+    text-align: center;
+    min-width: 20rem;
+`;
+const Knapp = styled(KnappBase)`
+    width: 100%;
+`;
+
+interface Props {
+    feilendeSystemer: Array<string>;
+}
+
+function FeilendeSystemerModal(props: Props) {
+    const [open, setOpen] = useState(props.feilendeSystemer.isNotEmpty());
+    const feilendeSystem = props.feilendeSystemer.map((it, index) => {
+        if (it === InformasjonElement.EGEN_ANSATT) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om Egen Ansatt.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.BANKKONTO) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om bankkonto.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.DKIF) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet digital kontaktinformasjon.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.PDL_GT) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om geografisk tilknytning.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.NORG_KONTAKTINFORMASJON) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet kontaktinformasjonen.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.NORG_NAVKONTOR) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om NAV-kontor.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.VEILEDER_ROLLER) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om veileders roller.</Normaltekst>
+                </li>
+            );
+        } else if (it === InformasjonElement.PDL_TREDJEPARTSPERSONER) {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om foreldre/barn/ektefelle/advokat/etc.</Normaltekst>
+                </li>
+            );
+        } else {
+            return (
+                <li key={index} className="blokk-xxs">
+                    <Normaltekst>Vi får ikke hentet informasjon om {it}.</Normaltekst>
+                </li>
+            );
+        }
+    });
+    return (
+        <Modal
```

Comment:
Er det muligens litt heftig å bruke en modal her? Det blir veldig in-your-face, selvom det man kanskje ikke har greid å hente ut er kontonummer ?